### ISSUE TITLE
Pagination fetches all Objects instead of just one page

### ISSUE THREAD

**Issue by breathTake** [typicode/json-server#1565]
For reference using the newest version "1.0.0-beta.1".

Im Fetching from my employee category with http://localhost:3000/employees?_page=1&_per_page=10, but the response gives mit all the employees available in the db. 

Or am i understanding its purpose wrong and it is meant to be client side pagination and no serverside pag?

Im Getting this data object:

```
{
    "first": 1,
    "prev": null,
    "next": 2,
    "last": 850,
    "pages": 850,
    "items": 850,
    "data": [
        850 entries...
      ]
```

I was expecting to get something like that:

```
[
   {
        Object1: {...}
   },
   {
        Object1: {...}
   }
   total 10...
]
```

**Comment by abdullateef97** [comment]
@breathTake have you resolved this

**Comment by breathTake** [comment]
> @breathTake have you resolved this

Yes, I downgraded to the last full release version, but I didn’t worked in this version unfortunately.

-> v0.17.4

**Comment by ivanzusko** [comment]
@breathTake, I believe it works as it should.

It returns:

```
{
   "first": 1,
  "prev": null,
  "next": 2,
  "last": 126,
  "pages": 126,
  "items": 1259,
  "data": ...//10 objects
```

You receive 10 items (as requested), from the 1st page.
The rest info:
`first` - first page to show in pagination (eg `1 ... 126`)
`next` - next page to request for pagination (if you click arrow)
`last` - the last page to show in your pagination (eg  `< 1 2 3 ... 126`)
`pages` - total amount of pages
`items` - total amount of items

**Comment by Irfan02x** [comment]
The current behavior seems to be working as intended.

The `_page` and `_per_page` parameters correctly limit the number of returned items, and the response includes pagination metadata along with the paginated data inside the `data` field.

From the example, the `data` array contains only the requested number of records (10), while fields like `first`, `next`, `pages`, and `items` provide useful pagination context.

If a plain array response (without metadata) is required, it might be worth considering either:

* a configuration flag (e.g., `_raw=true`), or
* clarifying this behavior in the documentation

to avoid confusion for new users.
